build/asset-manifest.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Hi Amy, AMAZING work on the website!! I was just wondering whether it's possible to consolidate the `build/` file under the `website/` directory, so that we only have one folder related to the website instead of 2?